### ISSUE TITLE
`php artisan backpack:crud` output is now messy

### ISSUE THREAD

**Issue by tabacitu** [Laravel-Backpack/Generators#134]
In the latest Laravel, they've prettified the artisan output. But now that makes our output messy:

<img width="533" alt="CleanShot 2022-07-20 at 12 57 45@2x" src="https://user-images.githubusercontent.com/1032474/179954684-cbdd21fc-2fa7-4db3-a372-09a63762bd8f.png">

We should stick to just one output type, so it's all easy-to-read and pretty too look at.

**Comment by tabacitu** [comment]
You run `php artisan backpack:crud user`. What do you see?

```
[INFO] Creating CRUD for the User model:

Creating App\Models\User ............................................................................. NOT NEEDED
Creating App\Http\Admin\UserCrudController ........................................................... DONE
Creating App\Http\Requests\UserRequest ............................................................... DONE
Adding CrudTrait to model ............................................................................ DONE
Adding route to routes/backpack/custom.php ........................................................... DONE
Adding sidebar entry to resources/views/vendor/backpack/inc/sidebar_content.blade.php ................ DONE

[INFO] Successfully created a CRUD for the User model.
| Go to https://your-project.test/admin/user to see it in action. 

```

**Comment by tabacitu** [comment]
Ping @promatik - this is priority numero uno right now. After we fix this and merge https://github.com/Laravel-Backpack/CRUD/pull/4577 - we have our fresh new onboarding flow 🎉

**Comment by tabacitu** [comment]
Solved in #136 🎉
